components/bl-transferList-component/src/index.js

```diff
@@ -0,0 +1,58 @@
+import { useMemo } from 'react';
+
+import { validate } from './utils/validate';
+import { useTransferList } from './hooks/useTransferList';
+
+import { List, TransferButtons } from './components';
+
+export default function TransferListComponent({ component, eventHandlers }) {
+  const {
+    display,
+    listType,
+    leftItems,
+    rightItems,
+    iconColor,
+    classList,
+  } = component;
+
+  const parsedLeftItems = validate(leftItems);
+  const parsedRightItems = validate(rightItems);
+  const [listState, dispatch] = useTransferList(parsedLeftItems, parsedRightItems);
+
+  if (!display) {
+    return null;
+  }
+
+  const classes = useMemo(() => {
```

Comment:
That doesn't trigger an error when we set display false on an already mounted component?
You could check it using Public API